src/streams/CreateStreamScreen.js

```diff
@@ -20,13 +22,19 @@ type Props = $ReadOnly<{|
 export default function CreateStreamScreen(props: Props): Node {
   const auth = useSelector(getAuth);
   const ownEmail = useSelector(getOwnEmail);
+  const getStreams = useSelector(getStreamsByName);
+  const allStreams = Array.from(getStreams.keys());
```

Comment:
The `getStreamsByName` selector gives a `Map<string, Stream>`, i.e., a [`Map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) where keys are strings and values are `Stream`s. As the name suggests, the keys are stream names.

`Map` gives you a way to see if there's a value at a given key, so you don't need to make an `Array` first: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/has

Reply:
Sure, @chrisbobbe!, Today evening, I got the idea of this i.e. `has` and kind of realised that, I need to change here 😉 and thankfully you pointed out.